### PR TITLE
Extract ima listener

### PR DESCRIPTION
**Pull request by dylanjha**
The Google IMA listener is being moved to a new pod. This PR removes all Google IMA stuff so that application developers that want to use `Mux-Stats-AVPlayer` don't have to also install the google ads library.

This branch can be tested now with a podfile update:

```
pod 'Mux-Stats-AVPlayer', :git => 'https://github.com/muxinc/mux-stats-sdk-avplayer', :branch => 'dj/extract-ima-listener'
```

However, after code review I would like to publish this to a beta tag, like `0.1.16.beta-0` before a production release.

TODO 

- [ ] create a useful README
- [ ] figure out Carthage deployment
- [ ] code review
- [x] publish beta tag and QA

### CODE REVIEW

File: MUXSDKStats/MUXSDKStats/Info.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.1.5</string>
+	<string>0.1.6-beta.0</string>
```
Comment (theRealWardo) [inline MUXSDKStats/MUXSDKStats/Info.plist]: 1.0.0

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.h [hunk 2]
```diff
@@ -24,8 +24,8 @@ typedef NS_ENUM(NSUInteger, MUXSDKPlayerState) {
 
 
 @interface MUXSDKPlayerBinding : NSObject {
-@private
     NSString *_name;
+@private
```
Comment (theRealWardo) [inline MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.h]: why did this move down?

File: MUXSDKStats/MUXSDKStats/MUXSDKStats.h [hunk 3]
```diff
@@ -64,9 +65,10 @@ FOUNDATION_EXPORT
  @param       name A name for this instance of the player
  @param       playerData A MUXSDKCustomerPlayerData object with player metadata
  @param       videoData A MUXSDKCustomerVideoData object with video metadata
+ @return      an instance of MUXSDKAVPlayerLayerBinding or nill
```
Comment (theRealWardo) [inline MUXSDKStats/MUXSDKStats/MUXSDKStats.h]: or null

File: MUXSDKStats/MUXSDKStats/MUXSDKStats.h [hunk 4]
```diff
@@ -104,14 +106,4 @@ FOUNDATION_EXPORT
...
 
-/*!
- @method      getImaAdsListener:
- @abstract    Return an IMA Ads listener for Ad analytic collection
- @param       name The name of the player
- @return      an IMA event listener
- @discussion  Use this method to return an IMA Ads listener to collect IMA Ad event and Ad error event.
- */
-#if TARGET_OS_IOS
-+ (MUXSDKImaListener *)getImaAdsListener:(nonnull NSString *)name;
```
Comment (theRealWardo) [inline MUXSDKStats/MUXSDKStats/MUXSDKStats.h]: ✌️

File: MUXSDKStats/MUXSDKStatsTv/Info.plist [hunk 5]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.1.5</string>
+	<string>0.1.6-beta.0</string>
```
Comment (theRealWardo) [inline MUXSDKStats/MUXSDKStatsTv/Info.plist]: 1.0.0

File: Mux-Stats-AVPlayer.podspec [hunk 6]
```diff
@@ -14,6 +14,8 @@ Pod::Spec.new do |s|
   s.license          = 'Apache 2.0'
   s.author           = { 'Mux' => 'ios-sdk@mux.com' }
 
+  s.dependency 'Mux-Stats-Core', '~>2.0.0'
```
[the remark this answers is not in the document]
Comment (theRealWardo) [reply Mux-Stats-AVPlayer.podspec]: very nice. I like it.

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.h (hunk 2 above)
Comment (dylanjha) [reply MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.h]: oh ya... that was me trying to expose the instance variable for debugging purposes

**Pull request merged**